services/web-app/pages/assets/components.js

```diff
@@ -26,6 +30,14 @@ const Components = ({ librariesData }) => {
   return (
     <>
       <NextSeo {...seo} />
+      <Grid className={styles.container}>
```

Comment:
We need this page header on multiple pages. Can you move this to a new `PageHeader` component, that we use in all ~6 catalog pages?

I'd expect the props for the PageHeader component would be `heading` as a string, and `icon` which we'd pass in a pictogram so each page could use a different pictogram.

Reply:
Can you add the page header to all catalog pages?